charts/datadog/templates/agent-clusterchecks-deployment.yaml

```
@@ -45,6 +45,7 @@ spec:
 {{ toYaml .Values.clusterChecksRunner.podAnnotations | indent 8 }}
       {{- end }}
     spec:
+      shareProcessNamespace: {{ .Values.clusterChecksRunner.shareProcessNamespace | default true }}
```

no need to provide a default value here since since a value is already present in the `values.yaml`

```suggestion
      shareProcessNamespace: {{ .Values.clusterChecksRunner.shareProcessNamespace }}
```